### PR TITLE
fix(paginator): delete number 1 from input

### PR DESCRIPTION
**Pull request by malegreIndec**
# Description

Allow the user to delete the number 1 of the input of type number

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How to test

1. List instructions for how to test
1. Make sure it's detailed and includes the exact steps

### CODE REVIEW

File: src/components/Pagination/Pagination.js [hunk 1]
```diff
@@ -58,6 +58,12 @@ const Pagination = ({
         }
     };
 
+    const handleBlur = () => {
```
Comment (arielerv) [inline src/components/Pagination/Pagination.js:61]: const handleBlur = () => Number.isNaN(search) && setSearch(1)

File: src/components/Pagination/Pagination.js [hunk 2]
```diff
@@ -115,8 +121,10 @@ const Pagination = ({
                                 min={1}
                                 max={totalPages}
                                 onChange={handleChangeInput}
```
Comment (arielerv) [inline src/components/Pagination/Pagination.js:123]: creo que el problema esta aca, cuando borras, le manda value = "", y como tiene un parseInt, no lo deja setea.. quizas deberia cambiar a:
const handleChangeInput = value => setSearch(Number(value) || "");
cuando Number(value) da 0.. va a ir por el or y se setea el empty string.

File: src/components/Pagination/Pagination.js (hunk 1 above)
Comment (malegreIndec) [reply src/components/Pagination/Pagination.js]: Buenas Ari, en el último commit mando los cambios que mencionaste en los comentarios

**Comment by maximilianoforlenza** [comment]
Fijate que fallo el job de eslint https://github.com/indec-it/react-commons/actions/runs/7386879380/job/20094364294

**Review by arielerv — APPROVED** [review]

**Pull request merged**